.github/workflows/trivy-all.yml

```
@@ -0,0 +1,46 @@
+name: Trivy Analysis (indy_node:latest)
+
+on:
+  schedule:
+    # run every Wednesday at 23:20
+    - cron: '20 23 * * 3'
+  workflow_dispatch:
+
+
+jobs:
+  trivy_analysis:
+    name: Trivy Analysis of Indy Node Images
+    runs-on: "ubuntu-20.04"
+    strategy:
+      matrix:
+        os_version: [ bullseye, buster, ubuntu16, ubuntu18]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Run Trivy on indy_node:${{ matrix.os_version }}
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: 'ghcr.io/idunion/indy-node-container/indy_node:main-${{ matrix.os_version }}'
```

```suggestion
          image-ref: 'ghcr.io/idunion/indy-node-container/indy_node:latest-${{ matrix.os_version }}'
```
I think we should check the latest released version here.

---

done.